atlp-rwanda/knights-ecomm-fe
# Feat-Admin-Dashbord-Users

What does this PR do?

- This PR introduces new features to the admin dashboard, including retrieving all users, viewing a single user, and activating or deactivating users.

Description of Task Completed

- Designed and integrated UI components for the admin dashboard to manage users.
- Implemented state management using Redux to maintain user data across sessions.
- Added functionality to retrieve and display all users in a paginated format.
- Created individual user pages to view detailed information about each user.
- Developed features to activate and deactivate users, providing admin control over user statuses.
- Ensured seamless integration with backend APIs for user operations, including fetching user data, activating, and deactivating users.
- Conducted unit tests to verify functionality and maintain code quality, ensuring robust performance under various scenarios.

How should this be manually tested?

- Ensure You have backend service running and set VITE_APP_API_URL in your .env file.
- Clone this repository and navigate to the project director
